mypy/semanal.py

```diff
@@ -2120,14 +2120,18 @@ def analyze_namedtuple_assign(self, s: AssignmentStmt) -> bool:
         """Check if s defines a namedtuple."""
         if isinstance(s.rvalue, CallExpr) and isinstance(s.rvalue.analyzed, NamedTupleExpr):
             return True  # This is a valid and analyzed named tuple definition, nothing to do here.
-        if len(s.lvalues) != 1 or not isinstance(s.lvalues[0], NameExpr):
+        if len(s.lvalues) != 1 or not (isinstance(s.lvalues[0], NameExpr) or
+                                       isinstance(s.lvalues[0], MemberExpr)):
```

Comment:
Style nit: The latter part could be written a bit more compactly as `isinstance(s.values[0], (NameExpr, MemberExpr))`.